std/experimental/allocator/common.d

```diff
@@ -407,6 +407,7 @@ Forwards each of the methods in `funs` (if defined) to `member`.
     return result;
 }
 
+version(unittest)
 package void testAllocator(alias make)()
```

Comment:
Imho `make` is a terrible name, sth. like `alloc` would be more intuitive.
For the lazy people it's used like this:

```
testAllocator!(() => FreeTree!GCAllocator());
testAllocator!(() => ScopedAllocator!GCAllocator());
...
```